packages/staking/src/features/drawer/StakePoolDetails.tsx

```diff
@@ -60,24 +69,32 @@ export const StakePoolDetails = ({
   );
 
   const footersMap = useMemo(
-    (): Record<DrawerStep, React.ReactElement> => ({
-      [DrawerDefaultStep.PoolDetails]: <StakePoolDetailFooter popupView={popupView} />,
-      [DrawerManagementStep.Preferences]: <StakePoolPreferencesFooter />,
+    (): Record<DrawerStep, React.ReactElement | null> => ({
+      [DrawerDefaultStep.PoolDetails]: (() => {
+        if (activeFlow !== Flow.PoolDetails || delegationPending || !selectionActionsAllowed) {
+          return null;
+        }
+        return <StakePoolDetailFooter popupView={popupView} />;
```

Comment:
I think it is even easier to understand if we invert the condition. And also it is worth to add a case with the CurrentPoolDetails to make it explicit
```suggestion
        if (activeFlow === Flow.CurrentPoolDetails || (activeFlow === Flow.PoolDetails && !delegationPending && selectionActionsAllowed) {
          return <StakePoolDetailFooter popupView={popupView} />;
        }
        return null;
```